builder/imports/llvm.py

```diff
@@ -121,6 +126,12 @@ def install(self, env):
         # Make script executable
         os.chmod(script_path, stat.S_IRUSR | stat.S_IRGRP |
                  stat.S_IROTH | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
-        sh.exec(*sudo, [script_path, version], check=True)
+
+        def _install_llvm():
+            sh.exec(*sudo, [script_path, version], check=True)
+
+        packages = UniqueList(config.get('compiler_packages', []))
```

Comment:
Shall we change data.py back to use compiler_packages instead of apt_compiler_packages?

Reply:
Nope. `apt_compiler_packages` gets munged down to `compiler_packages` on apt platforms.